library/src/skins/default/src/sass/tool.scss

```diff
@@ -255,9 +255,23 @@ label {
 	z-index: 1200 !important;
   }
 
-// basic-lti tool 
+// FontAwesome Icon Picker //
 .iconpicker-popover {
     opacity: 1 !important;
+    border-radius: 1px;
+}
+
+.iconpicker-popover .popover-title,
+.iconpicker-popover .iconpicker-items {
+    background-color: var(--sakai-background-color-2) !important;
```

Comment:
The usual question: Do you really need this !important on here.

Reply:
I should have mentioned it. We need it, or else Fontawesome would overwrite it.

Reply:
Makes perfect sense. Thanks.